pkgs/top-level/all-packages.nix

```diff
@@ -30941,8 +30941,8 @@ with pkgs;
 
   welle-io = libsForQt5.callPackage ../applications/radio/welle-io { };
 
-  wireshark = callPackage ../applications/networking/sniffers/wireshark {
-    inherit (darwin.apple_sdk.frameworks) ApplicationServices SystemConfiguration;
+  wireshark = darwin.apple_sdk_11_0.callPackage ../applications/networking/sniffers/wireshark {
```

Comment:
Why is it necessary to use a custom `callPackage` for darwin support?

Reply:
The point is that `x86_64-darwin` targets macOS 10.12, which is too old for Wireshark, so the macOS 11 SDK needs to be included somehow. `qt6Packages.callPackage` should also work here.